packages/fx-core/resource/package.nls.json

```diff
@@ -246,6 +246,11 @@
   "core.copilotPlugin.validate.openAIPluginManifest.summary": "Teams Toolkit has checked your OpenAI plugin manifest:\n\nSummary:\n%s.\n%s\n%s",
   "core.copilotPlugin.validate.summary.validate.failed": "%s failed",
   "core.copilotPlugin.validate.summary.validate.warning": "%s warning",
+  "core.copilotPlugin.scaffold.summary": "We have detected the following issues for your OpenAPI Specification file:\n%s",
+  "core.copilotPlugin.scaffold.summary.warning.operationId": "%s Mitigation: Not required, operationId has been automatically generated and added into %s file.",
+  "core.copilotPlugin.scaffold.summary.warning.teamsManifest.lengthExceeding": "%s must not have more than %s characters. ",
```

Comment:
please double check that the returned value for %s has double quotes, otherwise it is suggested to change 

%s -> "%s"

Reply:
Updated. 
![image](https://github.com/OfficeDev/TeamsFx/assets/86260893/3f868498-f7a9-4b83-8994-a23cf2a60786)